docker-compose/tail-sampling/jaeger-v2-config.yml

```
@@ -0,0 +1,45 @@
+service:
+  extensions: [jaeger_storage, jaeger_query, healthcheckv2]
+  pipelines:
+    traces:
+      receivers: [otlp]
+      processors: [tail_sampling]
+      exporters: [jaeger_storage_exporter]
+  telemetry:
+    logs:
+      level: DEBUG
+
+extensions:
+  healthcheckv2:
+    use_v2: true
+    http:
+  jaeger_query:
+    trace_storage: some_storage
+  jaeger_storage:
+    backends:
+      some_storage:
+        memory:
+          max_traces: 100000
+
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:4317
+
+processors:
+  tail_sampling:
+    decision_wait: 15s
```

do you need to wait 15s? tracegen generates traces very fast.
```suggestion
    decision_wait: 5s
```